tensorboard/plugins/beholder/BUILD

```diff
@@ -8,43 +8,77 @@ licenses(["notice"])  # Apache 2.0
 exports_files(["LICENSE"])
 
 py_library(
-    name = "beholder_plugin",
-    srcs = [
-        "beholder_plugin.py",
-        "shared_config.py",
+    name = "beholder",
+    srcs = ["beholder.py"],
+    data = ["resources"],
+    srcs_version = "PY2AND3",
+    deps = [
+        ":file_system_tools",
+        ":im_util",
+        ":shared_config",
+        ":video_writing",
+        ":visualizer",
+        "//tensorboard:expect_numpy_installed",
+        "//tensorboard:expect_tensorflow_installed",
+        "//tensorboard/backend/event_processing:plugin_asset_util",
```

Comment:
nit: Should `//tensorboard/backend/event_processing:plugin_asset_util` be a dep of `beholder_plugin` instead?

Reply:
It's a dep of both actually, but I removed it here on beholder.py since it's not currently used (although it probably should be, which is why I hadn't originally removed it).